src/coreclr/src/inc/corinfo.h

```diff
@@ -1782,9 +1782,15 @@ struct CORINFO_EH_CLAUSE
 enum CORINFO_OS
 {
     CORINFO_WINNT,
-    CORINFO_PAL,
+    CORINFO_UNIX,
 };
 
+#ifdef FEATURE_PAL
```

Comment:
We prefer to avoid ifdefs in corinfo.h. They tend to break various cross-compilation modes. It would be better to replicate this ifdef in the two places where `CORINFO_OS_HOST` is used.